docset/winserver2022-ps/defender/Get-MpPerformanceReport.md

```diff
@@ -0,0 +1,331 @@
+---
+description: Use this topic to analyze and report on Microsoft Defender Antivirus (MDAV) performance recordings.
+external help file: MSFT_MpThreat.cdxml-help.xml
+Module Name: Defender
+ms.date: 03/20/2022
+online version: 
+schema: 2.0.0
+title: Get-MpPerformanceReport
+
+---
+
+# Get-MpPerformanceReport
+
+The following section describes the Get-MpPerformanceReport PowerShell cmdlet. Analyzes and reports on Microsoft Defender Antivirus (MDAV) performance recording.
+
+## Syntax: Get-MpPerformanceReport
+
+```powershell
+Get-MpPerformanceReport    [-Path] <String>
+[-TopScans <Int32>]
+[-TopFiles  <Int32>
+    [-TopScansPerFile <Int32>]
+    [-TopProcessesPerFile  <Int32>  
+        [-TopScansPerProcessPerFile <Int32>]
+    ]
+] 
+[-TopExtensions  <Int32>
+    [-TopScansPerExtension <Int32>]
+    [-TopProcessesPerExtension <Int32>
+        [-TopScansPerProcessPerExtension <Int32>]
+        ]
+    [-TopFilesPerExtension  <Int32>
+        [-TopScansPerFilePerExtension <Int32>]
+        ]
+    ] 
+]
+[-TopProcesses  <Int32>
+    [-TopScansPerProcess <Int32>]
+    [-TopExtensionsPerProcess <Int32>
+        [-TopScansPerExtensionPerProcess <Int32>]
+    ]
+]
+[-TopFilesPerProcess  <Int32>
+    [-TopScansPerFilePerProcess <Int32>]
+]
+[-MinDuration <String>]
+```
+
+## Description: Get-MpPerformanceReport
+
+The `Get-MpPerformanceReport` cmdlet analyzes a previously collected Microsoft Defender Antivirus performance recording ([New-MpPerformanceRecording](#new-mpperformancerecording)) and reports the file paths, file extensions, and processes that cause the highest impact to Microsoft Defender Antivirus scans.
+
+The performance analyzer provides an insight into problematic files that could cause a degradation in the performance of Microsoft Defender Antivirus. This tool is provided "AS IS" and is not intended to provide suggestions on exclusions. Exclusions can reduce the level of protection on your endpoints. Exclusions, if any, should be defined with caution.
+
+For more information on the performance analyzer, see [Performance Analyzer](/windows-hardware/test/wpt/windows-performance-analyzer) docs.
+
+**Supported OS versions**
+
+Windows Version 10 and later.
+
+> [!NOTE]
+> This feature is available starting with platform version 4.18.2108.X and later.
+
+## Examples: Get-MpPerformanceReport
+
+### Example 1: Single query 
+
```

Comment:
You should have descriptions for each example. And sample output might be useful.